applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/adjoint_local_stress_response_function.cpp

```diff
@@ -207,7 +211,9 @@ namespace Kratos
     double AdjointLocalStressResponseFunction::CalculateNodeStress(ModelPart& rModelPart)
     {
         Vector element_stress;
-        mpTracedElement->Calculate(STRESS_ON_GP, element_stress, rModelPart.GetProcessInfo());
+
+        Element& r_element = rModelPart.GetElement(mpTracedElement->Id());
+        StressCalculation::CalculateStressOnGP(r_element, mTracedStressType, element_stress, rModelPart.GetProcessInfo());
```

Comment:
This should be ```CalculateStressOnNode```

Reply:
Was it wrong in the previous version of the file then? Because there in this function the `STRESS_ON_GP` was requested...

Reply:
You are right. I think this is a mistake in the previous version.

Reply:
fixed it